exasol_transformers_extension/upload_model.py

```diff
@@ -0,0 +1,55 @@
+import click
+from exasol_transformers_extension.utils import bucketfs_operations
+from exasol_transformers_extension.deployment import deployment_utils as utils
+
+
+@click.command()
+@click.option('--model-name', type=str, required=True,
+              help="name of the model")
+@click.option('--sub-dir', type=str, required=True,
+              help="directory where the model is stored in the BucketFS")
+@click.option('--model-path', type=click.Path(exists=True, file_okay=True),
+              required=True, help="local path where model is located")
+@click.option('--tokenizer-path', type=click.Path(exists=True, file_okay=True),
+              required=True, help="local path where tokenizer model is located")
+@click.option('--bucketfs-name', type=str, required=True)
+@click.option('--bucketfs-host', type=str, required=True)
+@click.option('--bucketfs-port', type=int, required=True)
+@click.option('--bucketfs_use-https', type=bool, default=False)
+@click.option('--bucketfs-user', type=str, required=True, default="w")
+@click.option('--bucketfs-password', type=str)
+@click.option('--bucket', type=str, required=True)
+@click.option('--path-in-bucket', type=str, required=True, default=None)
+def main(
+        bucketfs_name: str,
+        bucketfs_host: str,
+        bucketfs_port: int,
+        bucketfs_use_https: bool,
+        bucketfs_user: str,
+        bucketfs_password: str,
+        bucket: str,
+        path_in_bucket: str,
+        model_name: str,
+        sub_dir: str,
+        model_path: str,
+        tokenizer_path: str):
+
+    # get password
+    bucketfs_password = utils.get_password(
```

Comment:
I would use https://click.palletsprojects.com/en/8.1.x/options/#password-prompts in combination with the [envar](https://click.palletsprojects.com/en/8.1.x/options/#values-from-environment-variables) setting of the option click options.